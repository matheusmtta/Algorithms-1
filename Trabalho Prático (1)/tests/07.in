15 i
20 i
105 p
25 i
65 p
15 r
17 i
17 r
16 i
11 p
25 r
8 i
20 r
72 p
12 i
16 r
13 i
5 i
16 p
8 r
13 r
29 i
26 i
27 i
11 i
30 i
28 i
23 i
28 r
32 p
1 i
11 r
5 r
26 r
6 i
14 i
27 r
30 r
24 i
1 r
58 p
41 p
14 r
12 r
23 r
65 p
6 r
3 i
77 p
31 i
21 i
7 p
21 r
31 r
19 p
26 p
3 r
10 i
68 p
10 r
34 p
9 i
18 i
29 r
19 i
66 p
