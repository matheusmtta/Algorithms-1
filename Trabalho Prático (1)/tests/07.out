6
3
3
5
2
2
2
2
4
3
2
3
2
3
3
3
